### PR TITLE
[refactor](cloud) refactor copy into make logic clear

### PR DESCRIPTION
**Pull request by sollhui**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HHoflittlefish777** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/cloud/catalog/CloudEnvFactory.java [hunk 1]
```diff
@@ -183,6 +184,11 @@ public LoadManager createLoadManager(LoadJobScheduler loadJobScheduler) {
         return new CloudLoadManager(loadJobScheduler);
     }
 
+    public LoadManager createLoadManager(LoadJobScheduler loadJobScheduler,
```
Comment (yujun777) [inline fe/fe-core/src/main/java/org/apache/doris/cloud/catalog/CloudEnvFactory.java:187]: delete function at line 183:  createLoadManager(LoadJobScheduler loadJobScheduler)

File: fe/fe-core/src/main/java/org/apache/doris/cloud/load/CloudCopyJob.java [hunk 2]
```diff
@@ -1,106 +0,0 @@
-// Licensed to the Apache Software Foundation (ASF) under one
```
Comment (yujun777) [inline fe/fe-core/src/main/java/org/apache/doris/cloud/load/CloudCopyJob.java:1]: Maybe use CloudCopyJob is better then CopyJob ?  cause it's only use in cloud mode.

File: fe/fe-core/src/main/java/org/apache/doris/cloud/load/CloudLoadManager.java [hunk 3]
```diff
@@ -46,5 +83,189 @@ public long createLoadJobFromStmt(InsertStmt stmt) throws DdlException {
         return super.createLoadJobFromStmt(stmt);
     }
 
+    public LoadJob createLoadJobFromStmt(CopyStmt stmt) throws DdlException {
+        Database database = super.checkDb(stmt.getDbName());
+        long dbId = database.getId();
+        BrokerLoadJob loadJob = null;
+        if (Config.isCloudMode()) {
```
Comment (yujun777) [inline fe/fe-core/src/main/java/org/apache/doris/cloud/load/CloudLoadManager.java:90]: line 90 ~ 98 make simple, see line 74;

What's more CloudXX only call when Config.isCloudMode = true. So directory cloud/xxx should no appear Config.isCloudMode

File: fe/fe-core/src/main/java/org/apache/doris/common/proc/LoadProcDir.java [hunk 4]
```diff
@@ -66,7 +67,7 @@ public ProcResult fetchResult() throws AnalysisException {
         if (db == null) {
             loadJobInfos = loadManager.getAllLoadJobInfos();
         } else {
-            loadJobInfos = loadManager.getLoadJobInfosByDb(db.getId(), null, false,
+            loadJobInfos = ((CloudLoadManager) loadManager).getLoadJobInfosByDb(db.getId(), null, false,
```
Comment (yujun777) [inline fe/fe-core/src/main/java/org/apache/doris/common/proc/LoadProcDir.java:70]: Need check Config.isCloudMode() ?

File: fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadManager.java [hunk 5]
```diff
@@ -101,10 +97,9 @@
 public class LoadManager implements Writable {
     private static final Logger LOG = LogManager.getLogger(LoadManager.class);
 
-    private Map<Long, LoadJob> idToLoadJob = Maps.newConcurrentMap();
-    private Map<Long, Map<String, List<LoadJob>>> dbIdToLabelToLoadJobs = Maps.newConcurrentMap();
-    private LoadJobScheduler loadJobScheduler;
-    private CleanCopyJobScheduler cleanCopyJobScheduler;
+    protected Map<Long, LoadJob> idToLoadJob = Maps.newConcurrentMap();
```
Comment (yujun777) [inline fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadManager.java:100]: for line 100 ~ 102,   if they are only use in cloud mode ?  if so, should put these code into CloudLoadManager.

**Review by yujun777 — CHANGES_REQUESTED** [review]
need put cloud related operation in load/loadv2  into cloud/load

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/cloud/load/CloudLoadManager.java [hunk 6]
```diff
@@ -46,5 +83,188 @@ public long createLoadJobFromStmt(InsertStmt stmt) throws DdlException {
         return super.createLoadJobFromStmt(stmt);
     }
 
+    public LoadJob createLoadJobFromStmt(CopyStmt stmt) throws DdlException {
+        Database database = super.checkDb(stmt.getDbName());
+        long dbId = database.getId();
+        BrokerLoadJob loadJob = null;
+        ConnectContext context = ConnectContext.get();
```
Comment (yujun777) [inline fe/fe-core/src/main/java/org/apache/doris/cloud/load/CloudLoadManager.java:90]: line 90 ~ 96:   ((CloudSystemInfoService) Env.getCurrentSystemInfo()).waitForAutoStartCurrentCluster();

**Review by yujun777 — CHANGES_REQUESTED** [review]
need update

**Comment by HHoflittlefish777** [comment]
run buildall

**Review by yujun777 — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**